src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download \"%s\"</string>
+  <string name="error_syncing_format">Could not sync \"%s\</string>
+  <string name="error_validating_format">Could not validate \"%s\</string>
```

Comment:
Don't use quotation marks, they're different for any language, and I prefer to avoid them if possible.

Reply:
What are the placeholders here? I think the download one at least should be in quotes. Translators are meant to keep track of localization, but I get the point. Should be avoided unless it is an error to omit.

Reply:
"Error syncing F-Droid" ("Could not sync F-Droid"), "Error downloading NewPipe" ("Could not download NewPipe"), "Error validating NewPipe" ("Could not validate NewPipe")